spec/javascripts/govuk_publishing_components/analytics-ga4/ga4-page-views.spec.js

```diff
@@ -177,99 +177,99 @@ describe('Google Tag Manager page view tracking', function () {
       content.setAttribute('lang', 'wakandan')
       expected.page_view.language = 'wakandan'
 
-      GOVUK.analyticsGA4.pageViewTracker.sendPageView()
+      GOVUK.analyticsGA4.analyticsModules.PageViewTracker.init()
       expect(window.dataLayer[0]).toEqual(expected)
     })
 
     it('set incorrectly', function () {
```

Comment:
Should this be `sets` or `sets it`?

Reply:
The full sentence would be `Google Tag Manager page view tracking returns a pageview with a language set incorrectly`, so I think that's correct. Although it doesn't read very well within that function, granted.